app/src/main/res/values/strings.xml

```diff
@@ -381,6 +381,7 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="timezone">Timezone</string>
     <string name="status" translatable="false">Status: %1$s</string>
     <string name="user_detail_status" translatable="false">Status</string>
+    <string name="user_timezone_none" translatable="false">None</string>
```

Comment:
None is a translatable string.

Reply:
@filipedelimabrito added strings in respective languages and added Todo's